webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -1537,6 +1538,14 @@ function computeComplianceOK (complianceArray) {
     return { percent: 0, number: 0};
   }
 }
+
+function reportsSum (complianceArray) {
+  if (Array.isArray(complianceArray)) {
+    return complianceArray.reduce(function(total, value) { return total + reduce} )
```

Comment:
what happen for reduce when array size is 1? Does it correcty returns the value, or does it fails ?

Reply:
should be `total+value`
(and it's ok for size == 1, just checked)